### DISCUSSION TITLE
Skynet Telegram Group Created

### DISCUSSION THREAD

**Discussion by AnyWhereButHere** [cloudwu/skynet#1087] [Show and tell]
大家好！
由于某种原因原来的tg群（电报群）https://t.me/skynet123 不能使用了。
所以我建了一个新群大家可以自由的讨论Skynet相关的内容。
地址是：https://t.me/skynetgroup 
希望大家踊跃发言~

Hi everyone!
For some reson ,the old telegram group https://t.me/skynet123 no longer exists.
So i had Created a new telegram group for discuss everything about Skynet Freely.
the address is：https://t.me/skynetgroup 
Welcome for you!

**Comment by G0561** [comment]
你是不是天空， 建立辣椒，然后又跑路  ，为挺你，一同事和我打听跟天空买了3年辣椒，还有2年

**Comment by AnyWhereButHere** [comment]
不是。我也买了。

xiangheng <notifications@github.com>于2020年2月14日 周五22:05写道：

**Comment by cloudfreexiao** [comment]
我加进去了 好像没人
